### DISCUSSION TITLE
Texted messages

### DISCUSSION THREAD

**Discussion by Chrissy10** [community/community#37809] [Other Feature Feedback, Questions, & Ideas]
How can I stop emails and fake texted messages on my phone and gmail

**Comment by tuves** [comment]
Hi @Chrissy10,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this topic. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
